src/pages/Jieun/Login/Login.js

```diff
@@ -1,11 +1,86 @@
-import React from 'react'
-
-const Login = () => {
-    return (
-        <div>
-            
-        </div>
-    )
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
+
+import '../../../styles/reset.scss';
+import '../../../styles/common.scss';
+
+function Login() {
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
+  const [loginBtn, setLoginBtn] = useState(false);
+
+  const loginBtnChange = () => {
+    return id.includes('@') && pw.length > 4
+      ? setLoginBtn(true)
+      : setLoginBtn(false);
+  };
+
+  const navigate = useNavigate();
+  const goToMain = () => {
+    navigate('/jieun-main');
+  };
+
+  return (
+    <div className="loginJieun">
+      <main>
+        <img
+          className="loginImg"
+          alt="인스타그램"
+          src="/images/Jieun/login.png"
+        />
+        <section>
+          <div className="loginBox">
+            <div className="logo">westagram</div>
+            <input
+              className="id"
+              type="text"
+              placeholder="전화번호, 사용자 이름 또는 이메일"
+              onChange={e => setId(e.target.value)}
+              onKeyUp={loginBtnChange}
+            />
+            <input
+              className="pw"
+              type="password"
+              placeholder="비밀번호"
+              onChange={e => setPw(e.target.value)}
+              onKeyUp={loginBtnChange}
+            />
+            <div className="login">
+              <button
+                class={loginBtn ? 'loginOn' : 'loginOff'}
```

Comment:
className이라고 작성. 콘솔창에서 warning 발생했을 듯

Reply:
css에서 btnDefault를 mixin으로 만들어서 활용한 부분 좋았으나, 공통적인 css를 만들고 변하는 부분만 해당 className을 추가하는 방식으로 구현 가능
```suggestion
                className={`loginBtn ${isLoginValid ? '' : 'deactivate'}`}
```